egs2/chime4/asr1/local/data.sh

```diff
@@ -85,12 +85,6 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
     local/real_enhan_chime4_data_prep.sh beamformit_5mics ${PWD}/enhan/beamformit_5mics
     local/simu_enhan_chime4_data_prep.sh beamformit_5mics ${PWD}/enhan/beamformit_5mics
 
-    # prepare data for 6ch track:
-    #  (1) {tr05,dt05,et05}_simu_isolated_6ch_track
-    local/simu_ext_chime4_data_prep.sh --track 6 isolated_6ch_track ${PWD}/local/nn-gev/data/audio/16kHz
-    #  (2) {tr05,dt05,et05}_real_isolated_6ch_track
-    local/real_ext_chime4_data_prep.sh --track 6 isolated_6ch_track ${CHIME4}/data/audio/16kHz/isolated_6ch_track
-
```

Comment:
What is the intension here?

Reply:
In my last pull request, I added these lines for [chime4_asr_data.sh](https://github.com/espnet/espnet/blob/master/egs2/chime4/enh_asr1/local/chime4_asr_data.sh) in enh_asr1.
But, now I notice the same scripts are executed in [chime4_enh_data.sh](https://github.com/espnet/espnet/blob/master/egs2/chime4/enh_asr1/local/chime4_enh_data.sh) that is a symbolic link to `data.sh` in enh1.
So, they are redundnat and should be removed because `simu_ext_chime4_data_prep.sh` and `simu_ext_chime4_data_prep.sh` are not in `chime4/asr1/local`.
I'm really sorry for missing this point.

Reply:
Oh I see. Thanks for the clarification.